app/presenters/scoped_search_results_presenter.rb

```diff
@@ -0,0 +1,58 @@
+class ScopedSearchResultsPresenter < SearchResultsPresenter
+
+  def to_hash
+    super.merge({
+      is_scoped?: true,
+      scope_title: scope_title,
+      unscoped_results_any?: unscoped_results.any?,
+      unscoped_result_count: result_count_string(unscoped_result_count),
+      results: presentable_result_list,
+    })
+  end
+
+private
+
+  attr_reader :unscoped_results
+
+  def filter_fields
+    []
+  end
+
+  def scope_title
+    search_response["scope"]["title"]
+  end
+
+  def presentable_result_list
+    presentable_result_list = results
+
+    if unscoped_results.any?
+      insertion_point = results.count > 3 ? 3 : results.count
```

Comment:
`[results.count, 3].min` would work here, there's debate of course about which is more readable :)